frontend/src/scenes/surveys/SurveyView.tsx

```diff
@@ -528,7 +507,7 @@ export function SurveyResult({ disableEventsTable }: { disableEventsTable?: bool
 
     return (
         <div className="space-y-4">
-            <SurveyResultsFilters />
+            <SurveyAnswerFilters />
```

Comment:
Not sure if they are semantically different but it seems that the product has 'survey answers', 'survey responses', 'survey results', I think we should stick to one only if they are not different.
or at least eliminate the name duplication on docs and code

Reply:
I agree, let's go with `SurveyResponses` since our event is also `$survey_response` and that's not going to change